collectors/diskspace.plugin/plugin_diskspace.c

```diff
@@ -6,6 +6,7 @@
 
 #define DEFAULT_EXCLUDED_PATHS "/proc/* /sys/* /var/run/user/* /run/user/* /snap/* /var/lib/docker/*"
 #define DEFAULT_EXCLUDED_FILESYSTEMS "*gvfs *gluster* *s3fs *ipfs *davfs2 *httpfs *sshfs *gdfs *moosefs fusectl autofs"
+#define DEFAULT_EXCLUDED_FILESYSTEMS_INODES "msdosfs"
```

Comment:
This should also include:

- `msdos` and `vfat`: The equivalent to `msdosfs` on Linux.
- `overlayfs`, `aufs*`, `*unionfs`: These are all overlay filesystems, and it makes no sense for us to track inode counts for them.
- `*portal`: Used for the FUSE filesystems provided by https://github.com/flatpak/xdg-desktop-portal. These are virtual filesystems conceptually equivalent to GVFS, and this should actually also be added to the `DEFAULT_EXCLUDED_FILESYSTEMS` list.
- Everything in the `DEFAULT_EXCLUDED_FILESYSTEMS` list. If space tracking makes no sense, the inode tracking makes even less sense.
- Any read-only filesystem should be excluded both from inode tracking and space tracking if they are not already excluded.

Reply:
> Everything in the DEFAULT_EXCLUDED_FILESYSTEMS list. If space tracking makes no sense, the inode tracking makes even less sense.

Both `exclude space metrics on paths` and `exclude space metrics on filesystems` disable space **and** inode metrics.

Reply:
Added 1. and 2. to the default list.

Reply:
No, I am not going to address them in this PR.

Reply:
@thiagoftsm I’ll open a PR about the `fuse.portal` case myself (it’s an easy enough change, and I am actually able to test it locally), and I’ll start looking into whether the read-only case actually matters here or not.

Reply:
Correction, the `fuse.portal` case should be a non-issue, as that should always be mounted somewhere under `/run/user/*`, which we already exclude by default.